src/Numerics/GaussianBasisSet.h

```diff
@@ -15,19 +15,18 @@
 #ifndef QMCPLUSPLUS_RADIALGRIDFUNCTOR_GAUSSIANBASISSET_H
 #define QMCPLUSPLUS_RADIALGRIDFUNCTOR_GAUSSIANBASISSET_H
 #include <io/hdf_archive.h>
-#include "Numerics/OptimizableFunctorBase.h"
 #include "OhmmsData/AttributeSet.h"
 #include <cmath>
 #include "Message/CommOperators.h"
 namespace qmcplusplus
 {
 template<class T>
-struct GaussianCombo : public OptimizableFunctorBase
+struct GaussianCombo
 {
   // Caution: most other code assumes value_type can only be real
   // but maybe it can be different precision
   // Possibly one of these types is the full precision and the other reduced precision
-  typedef T value_type;
+  typedef T real_type;
```

Comment:
Later on this class appears as a template argument to Transform2GridFunctor and needs to discover T. Being able to see at a glance what is templated type in the class itself and that it is assumed real means I'd much prefer
```c++
template<typename REAL>
class GaussianCombo
{
 static_assert(std::is_floating_point<Real>::value,
                  "Other code assumes REAL is a real type");
 using Real = REAL; //Don't use this for member declarations;
...
```